src/context.js

```diff
@@ -94,7 +94,13 @@ export default (function () {
   function getState(key) {
     if (typeof store[key] === 'undefined') _initState(key, null);
     var listener = new ListenerConstructor(key);
-    return { value: store[key], onchange: listener };
+  
+    var setState = function(newValue) {
```

Comment:
i believe instead of passing a "setState" we should pass a "Listener" function as we were doing previously. the "setState" method for Astral.context is the Astral.context.updateState which the components can trigger from anywhere. The onchange listener here would allow the components to basically listen and react to changes. so we could do something like:

`return [store[key], listener];`

Reply:
Okay,  I'll make the adjustment and submit another PR